types/transaction.go

```diff
@@ -123,6 +123,21 @@ type TxData interface {
 	copy() TxData
 }
 
+func (tx *Transaction) String() string {
+	recipient := ""
+	if tx.To() != nil {
+		recipient = tx.To().String()
+	}
+
+	v, r, s := tx.RawSignatureValues()
+
+	return fmt.Sprintf("[%s] Nonce: %d, GasPrice: %d, GasTipCap: %d, GasFeeCap: %d, "+
```

Comment:
Should we print the access list as well?

Reply:
Fixed https://github.com/Ethernal-Tech/blade/pull/195/commits/79c0e49d286636fca7bec125db980ace70d842f5